man/CliquePercolation-package.Rd

```diff
@@ -6,10 +6,7 @@
 \alias{CliquePercolation-package}
 \title{CliquePercolation: Clique Percolation for Networks}
 \description{
-Clique percolation community detection for weighted and 
```

Comment:
CRAN requires lines to be short upon submission. Hence, the original line breaks need to stay.